### PR TITLE
feat: link component

### PR DESCRIPTION
**Pull request by annsch**

### CODE REVIEW

File: packages/foundations/scss/_init.scss [hunk 1]
```diff
@@ -42,6 +42,15 @@ a {
 	}
 }
 
+div,
+button,
+input,
+select,
+textarea {
+	font-family: inherit !important;
```
Comment (nmerget) [inline packages/foundations/scss/_init.scss:50]: Is there another solution intead of `!important`? :(

File: packages/components/src/components/link/model.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	href?: string;
+	anchorLink?: string;
+	title?: string;
+	rel?: string;
+	role?: string;
+	disabled?: boolean;
+	selected?: boolean;
+	label?: string;
+	hreflang?: string;
+	target?: string;
```
Comment (nmerget) [inline packages/components/src/components/link/model.ts:19]: Can we add all valid target options here: `_self`, `_blank`, ...

File: packages/components/src/components/link/model.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import {
+	ClickEventProps,
+	ClickEventState,
+	GlobalProps,
+	GlobalState
+} from '../../shared/model';
+
+export interface DBLinkDefaultProps {
```
Comment (nmerget) [inline packages/components/src/components/link/model.ts:8]: Do we need `referrerpolicy` -> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a?retiredLocale=de#attr-referrerpolicy

File: packages/components/src/components/link/link.lite.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import { onMount, Show, useMetadata, useStore } from '@builder.io/mitosis';
+import { DBIcon } from '../icon';
+import { DBLinkState, DBLinkProps } from './model';
+
+useMetadata({
+	isAttachedToShadowDom: true,
```
Comment (mfranzke) [inline packages/components/src/components/link/link.lite.tsx:6]: have we already decided on that we always want to use the ShadowDOM? I don't necessarily see any benefit on the elements.

File: packages/components/src/components/link/link.scss [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+	// "disabled" links
+	&[aria-disabled="true"] {
+		opacity: var(--db-disabled-text-opacity, 0.5);
+
+		pointer-events: none;
+
+		&::before {
+			color: initial;
+		}
```
Comment (mfranzke) [inline packages/components/src/components/link/link.scss:34]: what for do we need this declaration ?

File: packages/components/src/components/link/link.scss [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+	// Changing color on hover should be animated by a transition
+	transition: outline 0.06s, color $db-transition-color;
+
+	&:hover {
+		color: inherit;
+	}
+
+	// "disabled" links
+	&[aria-disabled="true"] {
+		opacity: var(--db-disabled-text-opacity, 0.5);
```
Comment (mfranzke) [inline packages/components/src/components/link/link.scss:28]:
```suggestion
		opacity: var(--db-disabled-text-opacity, $db-disabled-text-opacity);
```

There should most likely be a SCSS variable `$db-disabled-text-opacity` that we could use instead.

File: packages/components/src/components/link/link.scss (hunk 5 above)
Comment (mfranzke) [inline packages/components/src/components/link/link.scss:34]: what for do we need this declaration ?

```suggestion
```

File: packages/components/src/components/link/link.scss (hunk 6 above)
Comment (annsch) [reply packages/components/src/components/link/link.scss:28]: @mfranzke why do we need both? This is not a design token.

File: packages/foundations/scss/_init.scss (hunk 1 above)
Comment (annsch) [reply packages/foundations/scss/_init.scss]: 💯 just another copy & paste 🐙

File: packages/components/src/components/link/model.ts (hunk 3 above)
Comment (annsch) [reply packages/components/src/components/link/model.ts:8]: You're right, I missed that

File: packages/components/src/components/link/model.ts (hunk 2 above)
Comment (annsch) [reply packages/components/src/components/link/model.ts]: done

File: packages/components/src/components/link/link.lite.tsx (hunk 4 above)
Comment (annsch) [reply packages/components/src/components/link/link.lite.tsx]: changed that, I'm not sure if we could test the vanilla showcase already @nmerget

**Review by mfranzke — APPROVED** [review]

**Pull request merged**